frontend/components/droplets/lessons/generic-block-renderer.tsx

```
@@ -656,7 +667,7 @@ const GenericBlockRenderer: React.FC<GenericBlockRendererProps> = ({
           onMouseUp={() => handleMouseUp()}
           onMouseDown={(e) => handleMouseDown(e)}
           onClick={handleImageClick}
-          className="prose prose-lg prose-sky prose-table:block prose-code:text-inherit prose-table:overflow-x-scroll prose-p:my-1 prose-li:my-1 prose-headings:text-inherit prose-strong:text-inherit mt-2 select-text dark:text-slate-300"
+          className="prose prose-lg prose-sky prose-table:block prose-code:text-inherit prose-table:overflow-x-scroll prose-p:my-1 prose-li:my-1 prose-headings:text-inherit prose-strong:text-inherit select-text dark:text-slate-300"
           dangerouslySetInnerHTML={{ __html: nonTableContent }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)